resources/roadmap/index.md

```diff
@@ -187,20 +182,32 @@ permalink: /resources/roadmap/index.html
                                     <li class="completed">{% t roadmap.released-0-17 %}</li>
                             </ul>
                     </div>
-                    <label for="tab-6" tabindex="0"></label>
-                    <input id="tab-6" type="radio" name="tabs" aria-hidden="true" checked>
+                    <label for="tab-5" tabindex="0"></label>
+                    <input id="tab-5" type="radio" name="tabs" aria-hidden="true" checked>
                     <h2>2021</h2>
                     <div class="tabPanel-content info-block">
                         <h2>2021</h2>
                             <ul>
                                 <h3 class="months">{% t roadmap.feb %}</h3>
                                     <li class="completed">{% t roadmap.onionaddress %}</li>
                                 <h3 class="months">{% t roadmap.aug %}</h3>
-                                    <li class="completed">{% t roadmap.atomicswaps %}</li>
+                                    <li class="completed"><a href="{{ site.baseurl_root }}/2021/08/20/atomic-swaps.html" target="_blank" rel="noreferrer noopener">{% t roadmap.atomicswaps %}</a></li>
                                 <h3 class="months">{% t roadmap.oct %}</h3>
                                     <li class="completed"><a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">{% t roadmap.p2pool %}</a></li>
                             </ul>
                     </div>
+                    <label for="tab-6" tabindex="0"></label>
+                    <input id="tab-6" type="radio" name="tabs" aria-hidden="true" checked>
+                    <h2>2022</h2>
+                    <div class="tabPanel-content info-block">
+                        <h2>2022</h2>
+                            <ul>
+                                <h3 class="months">{% t roadmap.jul %}</h3>
+                                    <li class="completed"><a href="{{ site.baseurl_root }}/2022/07/19/monero-0.18.0.0-released.html" target="_blank" rel="noreferrer noopener">{% t roadmap.released-0-18 %}</a></li>
```

Comment:
Better use site.baseurl for consistency.

```suggestion
                                    <li class="completed"><a href="{{ site.baseurl }}/2022/07/19/monero-0.18.0.0-released.html">{% t roadmap.released-0-18 %}</a></li>
```

Reply:
Thanks, added changes. Shall i make the entire roadmap consistent with the use of  ```baseurl``` and not using ```target="_blank"``` when linking to a blog post?

Reply:
nah it doesn't worth it. There are probably a lot of inconsistencies on the website. We can change them when/if we touch those sections IMO.